schemas/com.snowplowanalytics.telemetry/collector_telemetry/jsonschema/1-0-0

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for collector telemetry",
+  "self": {
+    "vendor": "com.snowplowanalytics.telemetry",
+    "name": "collector_telemetry",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "collectorHost": {
+      "type": "string",
+      "description": "Host URL of the collector, e.g. https://collector.acme.com",
+      "format": "uri"
+    },
+    "collectorPath": {
+      "type": "string",
+      "description": "Path of the collector, e.g. /com.snowplowanalytics.snowplow/tp2"
+    },
+    "appId": {
+      "type": "string",
```

Comment:
As much as we like to pretend it isn't, app ID is technically optional so can we make this `"type": ["string", "null"]`?